subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -335,7 +335,7 @@ class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec imple
         run 'inputBrokenLinkNameCollector'
         then:
         executedAndNotSkipped ':inputBrokenLinkNameCollector'
-        output.text == "${[ADDED]}"
+        output.text == "[]"
```

Comment:
Shall we run this first without the symlink, and then add the broken link (and check that the task is skipped)?

Reply:
Sure, done.